### DISCUSSION TITLE
在某具体分类的页面显示相关标签的功能建议

### DISCUSSION THREAD

**Discussion by Deecyn** [volantis-x/hexo-theme-volantis#446] [Ideas]
有一个想法，就是在博客的某个具体分类的页面上，显示与该分类相关的标签。这里的相关是指：在发布文章的 front-matter 中，一同出现的分类和标签，就说他们是相关的。如下图所示：

![image](https://user-images.githubusercontent.com/33906598/90407028-8ceb2580-e0d8-11ea-87c3-d2b50857190a.png)

这样当博客文章较多的时候，可以更方便查。但是之前没看到过有这种功能的主题，可能是比较难实现吧～

PS: 博主这么好的主题，不考虑去其他平台上宣传一下吗😜

**Comment by xaoxuu** [comment]
这功能恐怕主题做不出来，你可以看看 hexo 和 next 社区有没有大佬做出来插件，有的话就是通用的。

**Comment by xaoxuu** [comment]
我不擅长这方面，segmentfault写过没几个人看。

**Comment by Reqwey** [comment]
网上找一个字符串模式匹配算法什么的？

**Comment by Reqwey** [comment]
仔细想想，其实不难，for一遍这个分类下的文章，把标签放出来不就行了

**Comment by Deecyn** [comment]
> 仔细想想，其实不难，for一遍这个分类下的文章，把标签放出来不就行了

我不熟悉前端，所以不太清楚，毕竟也没看见其它地方有这个功能。要不老哥你来实现一下？
